tslint.json

```diff
@@ -2,6 +2,7 @@
     "extends": ["tslint-microsoft-contrib"],
     "rulesDirectory": ["node_modules/tslint-microsoft-contrib", "node_modules/tslint-lines-between-class-members"],
     "rules": {
+        "align": false,
```

Comment:
Is this something we want to discuss before enforcing ? It seems nice to have a rule that aligns everything vertically. My 2c

Reply:
this rule conflicts with prettier. so disabling it.